### ISSUE TITLE
Multiple rules not supported?

### ISSUE THREAD

**Issue by hbea2014** [bee4/robots.txt#2]
I'm testing the parser and I noticed that I get strange results when there's more than one rule per user-agent. It looks like it's not supported (or maybe I'm not using the library correctly):

``` php
<?php

require __DIR__ . '/vendor/autoload.php';

use Bee4\RobotsTxt\Content;
use Bee4\RobotsTxt\ContentFactory;
use Bee4\RobotsTxt\Parser;

$content = new Content("
User-agent: *
Allow: /
Disallow: /private/page.html
Disallow: /private/

");

$rules = Parser::parse($content);

$result1 = $rules->match('*', '/private/page.html'); // returns true instead of false

$rule = $rules->get('*');

$result2 = $rule->match('/private/page.html'); // returns true instead of false
```

**Comment by shulard** [comment]
Hello,

Thanks for your feedback, I'll check it quickly!

**Comment by shulard** [comment]
I got it, but for the moment can't find a way to deal with it...

In fact it's relative to rule application order :
- I check if the URL is disallowed
- If yes, I check if it is allowed (because a URL can be disallowed by a generic rule but allowed by a more specific one)
- Here you `allow: /` so all the URL are allowed...

If you remove the `allow: /` from the robots.txt content it work... I commit a new branch to work on that issue (feature/issue-2). I hope I can deal with it before the end of the week.

I updated the PHPUnit test with this case (see: 7daaac8) so I'll be sure when it is resolved.
